packages/home-spa/src/_includes/partials/header.njk

```diff
@@ -9,7 +9,7 @@
   crossorigin="anonymous"
 ></script>
 <script
-  src="https://unpkg.com/@one-platform/opc-feedback@0.0.10-prerelease/dist/opc-feedback.js"
+  src="https://unpkg.com/@one-platform/opc-feedback@0.0.12-prerelease/dist/opc-feedback.js"
   integrity="sha384-AH+OgTmX8gp49M9W7oqGwMCVl98k1JBBGvpLfn80Ws5Z923vDa6HXKzQdBXQKb2u"
```

Comment:
```suggestion
  integrity="sha384-xeRtY4lUHm6jDOgB1DE6qKYBEzL3Nu30zvxWc0+BnVe4x7qyOHekVN9b83J0deSj"
```

SRI hash also needs to be updated
https://www.srihash.org/

Reply:
Updated accordingly